content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -371,18 +379,26 @@ When **not** annotating a field with the `?` type modifier, the field will be _r
 - **Database**: Required fields are represented via `NOT NULL` constraints in the underlying database.
 - **Prisma Client**: Prisma Client's generated [TypeScript types](#type-definitions) that represent the models in your application code will also define these fields as required to ensure they always carry values at runtime.
 
+### Unsupported types
+
+When you introspect a database, unsupported types are added as `Unsupported`:
```

Comment:
I'd maybe explain what unsupported types are. 

The `Unsupported` attribute acts as an escape hatch and allows you to define fields in the Prisma schema for database types that are not yet supported by Prisma. For example, MySQL's POLYGON type is not yet natively supported by Prisma, but can now be added to the Prisma schema using the Unsupported("polygon") type.

Note that fields of type Unsupported will not be surfaced in the Prisma Client API, however they can still be queried using the [`queryRaw`](/concepts/components/prisma-client/raw-database-access#queryraw) method.